src/proj.rs

```diff
@@ -792,16 +793,17 @@ impl Proj {
             .collect::<Result<Vec<_>, ProjError>>()?;
         pj.shrink_to_fit();
         // Transformation operations are slightly different
+        let pj_len = u64::try_from(pj.len()).unwrap();
         match op {
             Transformation::Conversion => unsafe {
                 proj_errno_reset(self.c_proj);
                 trans =
-                    proj_trans_array(self.c_proj, PJ_DIRECTION_PJ_FWD, pj.len(), pj.as_mut_ptr());
+                    proj_trans_array(self.c_proj, PJ_DIRECTION_PJ_FWD, pj_len, pj.as_mut_ptr());
                 err = proj_errno(self.c_proj);
             },
             Transformation::Projection => unsafe {
                 proj_errno_reset(self.c_proj);
-                trans = proj_trans_array(self.c_proj, inv, pj.len(), pj.as_mut_ptr());
+                trans = proj_trans_array(self.c_proj, inv, pj_len, pj.as_mut_ptr());
```

Comment:
https://github.com/rust-lang/rust-bindgen/pull/1688

Reply:
Won't this break against our prebuilt bindings?

I admit - I'm kind of surprised by the changes made to `bindgen`.

Reply:
I suppose we can just re-generate all the prebuilt bindings with the new bindgen.

Reply:
🤦 Oh yeah, I never did https://github.com/georust/proj/issues/44, so we don't have any prebuilt bindings. 😅

Reply:
> I admit - I'm kind of surprised by the changes made to `bindgen`.

there appears to be an ongoing discussions about that change https://github.com/rust-lang/rust-bindgen/issues/1901